_data/navigation.yml

```diff
@@ -10,39 +10,39 @@ main:
   - title: "Git Out of Trouble"
     beta: true
     url: /git-trouble/
-  - title: "Training Path"
+  - title: "Git & GitHub Training Path"
     url: /path/
-  - title: "Community"
+  - title: "GitHub Training Community"
     url: /community/
 
 # When creating new sidebar navs, remember the url in this document needs to match the url assigned in the permalink field.
 
 intro-nav:
-  - title: "Welcome"
+  - title: "Introduction to GitHub"
     url: /intro-to-github/
   - title: "Create a GitHub Account"
     url: /intro-to-github/01/
   - title: "Join the Class Repository"
     url: /intro-to-github/02/
-  - title: "Explore the Repository"
+  - title: "Explore the GitHub Repository"
     url: /intro-to-github/03/
   - title: "Use GitHub Flow"
     url: /intro-to-github/04/
   - title: "Create a Branch"
     url: /intro-to-github/05/
   - title: "Find Your Coordinates"
     url: /intro-to-github/06/
```

Comment:
@brianamarie did the SEO experts also make the point that we should use more than just 06/ for our perma links? I believe they were saying our URLs should also be descriptive??